src/chrome/content/rules/ZeniMax.com.xml

```diff
@@ -0,0 +1,39 @@
+<!--
+	Other ZeniMax Media rulesets:
+
+		- Arkane-Studios.com.xml
+		- Bethsoft.com.xml
+		- ElderScrolls.com.xml
+		- Elder_Scrolls_Online.com.xml
+		- Dishonored.com.xml
+		- Id-Software.com.xml
+		- ZeniMaxOnline.com.xml
+
+
+	Login required:
+		ausvpn.zenimax.com
+
+	Invalid certificate:
+		cdn.zenimax.com
+		static.zenimax.com
+
+	No working URL known:
+		lyncwebext.zenimax.com
+		meet.zenimax.com
+
+-->
+<ruleset name="ZeniMax.com">
+
+	<target host="zenimax.com" />
+	<target host="www.zenimax.com" />
+	<target host="dialin.zenimax.com" />
+	<target host="download.zenimax.com" />
+	<target host="jobs.zenimax.com" />
+	<target host="static.zenimax.com" />
+
+	<rule from="^http://static\.zenimax\.com/"
+		to="https://d1z7bw3wiqzsm4.cloudfront.net/" />
```

Comment:
@J0WI, I don't have any. I only left this rule because it was already there and I have no better way to secure this subdomain but I am fine with removing it (or simply commenting it out hoping someone will eventually come up with a test url).